test/modules/aws-account-integration.ts

```diff
@@ -14,8 +14,8 @@ const query = runQuery.bind(null, dbAlias);
 const sync = runSync.bind(null, dbAlias);
 
 jest.setTimeout(360000);
-beforeAll(execComposeUp);
-afterAll(() => execComposeDown());
+beforeAll(async () => await execComposeUp());
+afterAll(async () => await execComposeDown());
```

Comment:
In fact, since it doesn't use any of the input arguments either, wrapping them in anonymous functions is also not necessary. You can just write this and it should work fine:

```suggestion
beforeAll(execComposeUp);
afterAll(execComposeDown);
```

Reply:
this doesn't work

Reply:
Oh, that can explain that. They must be inspecting the type of the argument passed to it and doing different things for different types? :(